src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysQl://localhost:3306/test";

```

Comment:
There is a typo in the database URL. The correct protocol should be 'jdbc:mysql://' instead of 'jdbc:mysQl://'. This will cause a connection failure if not corrected.

Reply:
ok, done